iphone/Classes/TiUIAlertDialogProxy.m

```diff
@@ -134,7 +136,7 @@ -(void)show:(id)args
                 NSString* btnName = [TiUtils stringValue:btn];
                 if (!IS_NULL_OR_NIL(btnName)) {
                     UIAlertAction* theAction = [UIAlertAction actionWithTitle:btnName
-                                                                        style:((curIndex == cancelIndex) ? UIAlertActionStyleCancel : UIAlertActionStyleDefault)
+                                                                        style:((curIndex == cancelIndex) ? UIAlertActionStyleCancel : (curIndex == destructiveIndex) ? UIAlertActionStyleDestructive :UIAlertActionStyleDefault)
```

Comment:
This line is a bit confusing and not easy to follow due to the multiple ternary operators. If would be better to spread the out for clarity purposes, something like this:
```
UIAlertActionStyle style;
if (curIndex == cancelIndex) {
	style = UIAlertActionStyleCancel;
} else if (curIndex == destructiveIndex) {
	style = UIAlertActionStyleDestructive;
} else {
	style  = UIAlertActionStyleDefault;
}
```